app/views/firms/partials/_firm_details.html.erb

```diff
@@ -96,3 +96,17 @@
     </div>
   </div>
 </div>
+
+<div class="l-firm__row">
```

Comment:
Does `firm__divider` need to be its own thing? Perhaps it could just be a modifier on the `l-firm__row` itself? E.g. `l-firm__row--with-divider` or something